Update containerized install guide (#1851)
ansible/aap-docs: downstream/modules/platform/proc-downloading-containerized-aap.adoc

.Procedure

. Download the latest installer .tar file from the link:{PlatformDownloadUrl}[{PlatformNameShort} download page]. 

. Copy the installer .tar file and the optional manifest .zip file onto your {RHEL} host.

> . Decide where you want the installer to reside on the file system. Installation related files will be created under this location and require at least 10 GB for the initial installation.

Reviewer: ```suggestion
. Decide where you want the installer to reside on the file system. Installation related files are created under this location and require at least 10 GB for the initial installation.
```

. Copy the installer .tar file and the optional manifest .zip file onto your {RHEL} host.

> . Decide where you want the installer to reside on the file system. Installation related files will are created under this location and require at least 10 GB for the initial installation.

Reviewer: ```suggestion
. Decide where you want the installer to reside on the file system. Installation related files are created under this location and require at least 10 GB for the initial installation.
```
